src/unitxt/loaders.py

```diff
@@ -394,3 +395,27 @@ def process(self):
             )
 
         return MultiStream.from_iterables(dataset)
+
+
+class LoadHFCustomDatasetScript(Loader):
+    """Loads a dataset using a custom HF dataset loading script from catalog/loaders path."""
+
+    file: str
+
+    def process(self):
+        catalog_path = constants.__getattr__("local_catalog_path")
+
+        assert (
+            catalog_path is not None
+        ), "Please set the local_catalog_path in constants file"
+
+        script_path = os.path.join(catalog_path, "loaders", self.file)
+
+        assert (
+            os.path.exists(script_path) is True
+        ), "Script file does not exist in the local catalog path"
+
+        loader = LoadHF(path=script_path, streaming=False)
+        ms = loader.process()
+        dataset = ms.to_dataset()
+        return MultiStream.from_iterables(dataset)
```

Comment:
Why not?
```suggestion
        return  LoadHF(path=script_path, streaming=self.streaming)
```

Reply:
Tried this way of calling LoadHF directly first but it caused following error 
`Traceback (most recent call last):
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/prepare/cards/tabfact.py", line 49, in <module>
    test_card(card)
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/test_utils/card.py", line 260, in test_card
    test_with_eval(
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/test_utils/card.py", line 160, in test_with_eval
    examples = load_examples_from_standard_recipe(
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/test_utils/card.py", line 78, in load_examples_from_standard_recipe
    examples = print_recipe_output(
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/test_utils/card.py", line 134, in print_recipe_output
    message, examples = construct_recipe_output_message(
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/test_utils/card.py", line 104, in construct_recipe_output_message
    multi_stream = recipe()
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/operator.py", line 478, in __call__
    return super().__call__()
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/operator.py", line 157, in __call__
    result = self._process_multi_stream(multi_stream)
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/operator.py", line 168, in _process_multi_stream
    result = self.process(multi_stream)
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/operator.py", line 486, in process
    multi_stream = operator(multi_stream)
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/operator.py", line 157, in __call__
    result = self._process_multi_stream(multi_stream)
  File "/Users/rajmohan/Projects/EvalFramework/gitrepos/unitxt/src/unitxt/operator.py", line 202, in _process_multi_stream
    for stream_name, stream in multi_stream.items():
AttributeError: 'LoadHF' object has no attribute 'items'`

Then used the method followed in test_loaders.py as above which worked.